index.ts

```diff
@@ -614,13 +614,17 @@ class ResembleHelper extends Helper {
    * @returns {Promise<{ignoredBoxes: [{left: *, top: *, right: *, bottom: *}]>}
    */
   private async _reCountElementCoordinatesForIgnoreInScreenshotElement(selector: Selector, ignoredElementsCoordinates: BoxCoordinates[]): Promise<BoxCoordinates[]> {
+    const helper = this._getHelper();
     const boundingBox = await this._getBoundingBox(selector);
 
+    const scrollOffset = await helper.executeScript(() => ({ X: window.pageXOffset, Y: window.pageYOffset }));
```

Comment:
Did you test it for both WebDriver and Playwright?

Reply:
executeScript is a standard way for both drivers/helpers, this was tested in the past and worked good.